SoftLayer/managers/ordering.py

```diff
@@ -370,22 +372,28 @@ def get_price_id_list(self, package_keyname, item_keynames, core=None):
                   keynames in the given package
 
         """
-        mask = 'id, description, capacity, itemCategory, keyName, prices[categories]'
+        mask = 'id, description, capacity, itemCategory, keyName, prices[categories], \
+                softwareDescription[id,referenceCode,longDescription]'
```

Comment:
Doing the following will remove the extra whitespace from this mask, and should generally fix the other unit tests you have. 
```suggestion
        mask = 'id, description, capacity, itemCategory, keyName, prices[categories], ' \
                'softwareDescription[id,referenceCode,longDescription]'
```

These ones anyway.
```python
>           raise AssertionError(_error_message()) from cause
E           AssertionError: expected call not found.
E           Expected: list_items('PACKAGE_KEYNAME', mask='id, description, capacity, itemCategory, keyName, prices[categories], softwareDescription[id,referenceCode,longDescription]')
E           Actual: list_items('PACKAGE_KEYNAME', mask='id, description, capacity, itemCategory, keyName, prices[categories],                 softwareDescription[id,referenceCode,longDescription]')
```